Content.Client/Entry/EntryPoint.cs

```diff
@@ -121,6 +121,7 @@ public override void Init()
             _prototypeManager.RegisterIgnore("wireLayout");
             _prototypeManager.RegisterIgnore("alertLevels");
             _prototypeManager.RegisterIgnore("nukeopsRole");
+            _prototypeManager.RegisterIgnore("stationGoal"); // Corvax-StationGoal
```

Comment:
If we're going to take this and build upon it ourselves, it *probably* makes sense to just label it Parkstation for simplicity, no?